### PR TITLE
Initial commit to add hover effect on parks layer

### PR DESCRIPTION
**Pull request by edellucien**
# How to

- Hover effect for parks layer

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/layers/ParksLayer/ParksLayer.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    return false;
+  }
+
+  getFeatureInfoAtCoordinate(coordinate) {
+    this.abortController.abort();
+    this.abortController = new AbortController();
+    const { signal } = this.abortController;
+    return fetch(this.getFeatureInfoUrl(coordinate), { signal })
+      .then((resp) => resp.json())
+      .then((r) => r.features)
```
Comment (oterral) [inline src/layers/ParksLayer/ParksLayer.js:67]: r?

File: src/layers/ParksLayer/ParksLayer.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            });
+            multipolygon.push([polygon]);
+          });
+          const newFeature = new Feature({
+            geometry: new MultiPolygon(multipolygon),
+          });
+          newFeature.setId(feat.id);
+          newFeature.setProperties(feat.properties);
+          return newFeature;
+        });
```
Comment (oterral) [inline src/layers/ParksLayer/ParksLayer.js:89]: You can do better than this. Probably in 1 or 2 lines of code.

Comment (oterral) [reply src/layers/ParksLayer/ParksLayer.js:89]: GeoJSON FeatureCollection to an  Array<ol.Feature>

File: src/layers/ParksLayer/ParksLayer.js (hunk 1 above)
Comment (edellucien) [reply src/layers/ParksLayer/ParksLayer.js:67]: It comes from mobility-toolbox-js WMSLayer 'getFeatureInfoAtCoordinate' function.
r stands for response I think but I can use a clearer name :)

File: src/layers/ParksLayer/ParksLayer.js (hunk 2 above)
Comment (edellucien) [reply src/layers/ParksLayer/ParksLayer.js:89]: I initially tried with: https://openlayers.org/en/latest/apidoc/module-ol_format_GeoJSON-GeoJSON.html#readFeatures
But didn't made it work properly, I will try again :)

Comment (edellucien) [reply src/layers/ParksLayer/ParksLayer.js]: Shorten it with:
`data.map((d) => this.format.readFeature(d));`

File: src/layers/ParksLayer/ParksLayer.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      .then((response) => response.features)
+      .then((data) => {
+        const dataFeats = data.map((d) => this.format.readFeature(d));
+        this.vectorLayer.getSource().clear();
+        this.vectorLayer.getSource().addFeatures(dataFeats);
+
+        return {
+          layer: this,
+          coordinate,
+          features: data.map((d) => this.format.readFeature(d)),
```
Comment (oterral) [inline src/layers/ParksLayer/ParksLayer.js:80]: Review again ,

**Pull request merged**